maat-court-data-api/src/main/java/gov/uk/courtdata/entity/QueueMessageLogEntity.java

```diff
@@ -19,6 +19,9 @@ public class QueueMessageLogEntity {
     @Id
     @Column(name = "TRANSACTION_UUID")
     private String transactionUUID;
+    @Column(name = "LAA_TRANSACTION_ID")
```

Comment:
I guess we'll need to run an update script on the other envs to add in this column?

Reply:
Deployed to Dev, I'll update the database as per the code progressed through the CI until STG.